data/example_realization_config.json

```diff
@@ -26,14 +26,13 @@
                 "giuh_path": "./test/data/giuh/GIUH.json",
                 "crosswalk_path": "./data/crosswalk.json"
             }
-        },
-        "forcing": {
```

Comment:
Okay, I will put it back in. I took it out because the each catchment had its own forcing definition, and they weren't using this global one. I see now with Chris's latest merge that he is putting these global forcing vars to use.